src/MoBi.Presentation/Presenter/SpaceDiagram/SpatialStructureDiagramPresenter.cs

```diff
@@ -1,44 +1,50 @@
 using System.Drawing;
 using System.IO;
 using System.Linq;
-using OSPSuite.Core.Services;
-using OSPSuite.Utility.Container;
-using OSPSuite.Utility.Extensions;
 using MoBi.Core;
 using MoBi.Core.Domain.Model;
 using MoBi.Core.Services;
-using MoBi.Presentation.Settings;
 using MoBi.Presentation.Presenter.BaseDiagram;
+using MoBi.Presentation.Settings;
 using MoBi.Presentation.Tasks.Interaction;
 using MoBi.Presentation.Views.BaseDiagram;
 using OSPSuite.Core;
 using OSPSuite.Core.Diagram;
 using OSPSuite.Core.Domain;
 using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Services;
 using OSPSuite.Presentation.Diagram.Elements;
 using OSPSuite.Presentation.Presenters;
+using OSPSuite.Utility.Container;
+using OSPSuite.Utility.Extensions;
 using IContainer = OSPSuite.Core.Domain.IContainer;
 
 namespace MoBi.Presentation.Presenter.SpaceDiagram
 {
    public interface ISpatialStructureDiagramPresenter : IMoBiBaseDiagramPresenter<IMoBiSpatialStructure>, IPresenter<ISpatialStructureDiagramView>
    {
-
    }
 
    public class SpatialStructureDiagramPresenter : MoBiBaseDiagramPresenter<ISpatialStructureDiagramView, ISpatialStructureDiagramPresenter, IMoBiSpatialStructure>, ISpatialStructureDiagramPresenter
    {
       private readonly IMoBiConfiguration _configuration;
 
-      public SpatialStructureDiagramPresenter(ISpatialStructureDiagramView view, IContainerBaseLayouter layouter, IUserSettings userSettings, 
-         IMoBiContext context, IDialogCreator dialogCreator, IMoBiConfiguration configuration, IDiagramTask diagramTask, IStartOptions runOptions, 
-         IDiagramModelFactory diagramModelFactory, OSPSuite.Utility.Container.IContainer container)
-         : base(view, layouter, dialogCreator,diagramModelFactory, userSettings, context, diagramTask, runOptions, container)
+      public SpatialStructureDiagramPresenter(
+         ISpatialStructureDiagramView view,
+         IContainerBaseLayouter layouter,
+         IUserSettings userSettings,
+         IMoBiContext context,
+         IDialogCreator dialogCreator,
+         IMoBiConfiguration configuration,
+         IDiagramTask diagramTask,
+         IStartOptions runOptions,
+         IDiagramModelFactory diagramModelFactory)
+         : base(view, layouter, dialogCreator, diagramModelFactory, userSettings, context, diagramTask, runOptions)
       {
          _configuration = configuration;
-         _diagramPopupMenu = new PopupMenuSpaceDiagram(this, runOptions, container);
-         _containerPopupMenu = new PopupMenuFullContainerWithParametersNode(this, _context, runOptions, container);
-         _neighborhoodPopupMenu = new PopupMenuFullEntityNode<INeighborhoodBuilder>(this, _context, runOptions, container);
```

Comment:
@rwmcintosh I am removing those container that you added to each popup menu to satisfy the requirement of not using IoC. The container was already in the Context (not exposed) but I am exposing it now so that I don't have to have those weird `OSPSuite.Utility.Container.IContainer container` interfaces everywhere